README.md

```diff
@@ -65,9 +65,28 @@ npm install --save-dev babel-plugin-inline-react-svg
     ]
   ]
 }
+```
+- *`alias`* - An object with alias for module resolution
+- *`root`* - A relative path string (Starting from CWD), it only works in conjunction with alias.
```

Comment:
I'm confused why you need this, can't you just define the full paths you care about in the alias?

Reply:
You mean the root? I can change it to work with out, so its optional.

Reply:
I'm mostly confused why you need this at all, can't the aliases include the fully qualified path?

Reply:
@kesne If I may to add my 5 cents: [babel-plugin-module-resolver](https://www.npmjs.com/package/babel-plugin-module-resolver) uses `root` to specify the common prefix for all aliases, which is convenient when config and paths you alias are located in different places (otherwise, root should default to CWD). Moreover, `root` there accepts an array of paths, or glob path: this way it allows to look for aliases in multiple directories.

Reply:
It seems like we should allow `root` to be defined always then, not just in conjuction with alias.

Reply:
But without `alias` it is not supposed to do anything, I guess.

Reply:
@kesne @birdofpreyru Then what should be the best option here? . Indeed `babel-plugin-module-resolver` uses root, I believe I added for compatibility reasons, thought that module uses an array as mentioned before. As @birdofpreyru mentioned, the root property is useless without any aliases. If I'm not wrong the other plugin requires that you always specify the root, right now it works like this too, so I'm not sure now if we should change this or not, let me know what you think.